### ISSUE TITLE
USA scenario analysis: not show line in S-R trend analysis

### ISSUE THREAD

**Issue by lisphilar** [lisphilar/covid19-sir#121]
Summary:
With USA data, the following codes do not show fitting line in S-R trend analysis.

```Python
import covsirphy as cs
# Dataset preparation
data_loader = cs.DataLoader("input")
jhu_data = data_loader.jhu()
population_data = data_loader.population()
scenario = cs.Scenario(jhu_data, population_data, country="US")
scenario.trend()
```

Environment:
Python 3.8, pipenv, WSL.

**Comment by lisphilar** [comment]
In the next version, lines will not be shown when RMSLE scores is too large. Maximum value of RMSLE score will be set with `Scenario.trend(max_rmsle=20)`, and default value is 20.

**Comment by lisphilar** [comment]
This was fixed in version 2.5.4
